package.json

```diff
@@ -162,6 +164,8 @@
     "@react-native-community/cli": "^1.1.0",
     "art": "^0.10.0",
     "base64-js": "^1.1.2",
+    "clang-format": "^1.2.4",
+    "connect": "^3.6.5",
```

Comment:
This was me during the merge. `connect` was removed in another diff accidentally, and this diff still had it as part of the conflict, and I need to patch up `yarn.lock` anyway so I figured I'll just do it here. I realize this is not the proper way of dealing with it.